### PR TITLE
Rename tutorials to use cases

### PR DESCRIPTION
**Pull request by mheap**
## Description

Rename Tutorial to Use Case

The homepage still has `Tutorial` links. Once we update to have tutorials we'll switch where these links point to.

This is part 1 of a series of changes where we'll convert the existing "Send X with Y" tutorials that currently link to the blog to show up under Tutorials rather than Use Cases.

This PR removes two landing pages which are currently erroring in production as they were causing a new test that was added to fail

## Deploy Notes

N/A

### CODE REVIEW

**Comment by mheap** [comment]
Please review carefully and approve if you're happy, but don't merge

**Comment by mheap** [comment]
@fabianrbz @benhayehudi I'd love a code review on this one

**Comment by fabianrbz** [comment]
@mheap I'm currently reviewing it :D

File: app/filters/use_case_list_filter.rb [hunk 1]
```diff
@@ -1,14 +1,14 @@
-class TutorialsFilter < Banzai::Filter
+class UseCaseListFilter < Banzai::Filter
   def call(input)
-    input.gsub(/```tutorials(.+?)```/m) do |_s|
+    input.gsub(/```use_cases(.+?)```/m) do |_s|
       config = YAML.safe_load($1)
       @product = config['product']
-      @tutorials = Tutorial.by_product(@product)
+      @tutorials = UseCase.by_product(@product)
```
Comment (fabianrbz) [inline app/filters/use_case_list_filter.rb]: should we rename all the instance variables to `@tutorials`?
There are a lot of references to tutorials like [this one](https://github.com/Nexmo/nexmo-developer/blob/rename-tutorials-to-use-cases/app/views/use_case/_index.html.erb#L5) under `app/views/use_case`.

**Comment by mheap** [comment]
Comments from Fabian addressed. I think this is good to go. I'll merge at 11am UK time tomorrow and post in Slack when I do. Please keep an eye out for any issues when it goes live

**Pull request merged**